### ISSUE TITLE
Respect `jsxSingleQuote`

### ISSUE THREAD

**Issue by Sxxov** [sveltejs/prettier-plugin-svelte#253]
Currently the plugin formats single quotes in html markup into double quotes regardless if `jsxSingeQuote` is set. It would be nice if the plugin would respect the setting & use single quotes in html as well.

Source:
```html
<div class='arrow'></div>
```

Current behaviour:
```html
<div class="arrow"></div>
```

Expected behaviour:
```html
<div class='arrow'></div>
```

**Comment by svenjacobs** [comment]
I'm having the same issue that quotes in HTML are converted to `"` although `singleQuote` and `jsxSingleQuote` are set to `true`.

**Comment by svenjacobs** [comment]
I just had a quick look at the source code. I'm absolutely unfamiliar how Prettier or this plugin work. Maybe [this](https://github.com/sveltejs/prettier-plugin-svelte/blob/0473f025efb23d278dd2ac0c1865932134b587e4/src/print/index.ts#L393) is the faulty line?

**Comment by dummdidumm** [comment]
Yes this is probably the line (or one of the lines). The problem is that we would need to check the contents of attributes for quotes and switch them if necessary. I suspect there are quite a few people who like to use single quotes in their script tags but double quotes for attributes because that's more common for HTML. These people would get a breaking change experience because
- if we use `singleQuote` for attributes, too, they suddenly are formatted differently
- it might very well make code invalid if we don't switch quotes within the attribute (`attr="some { value ? 'bar' : 'foo'}"` -> `attr='some { value ? 'bar' : 'foo'}'` <- invalid)

So if we would add support for that then I think it should be a new setting, and it also shouldn't be `jsxSingleQuote` because Svelte has nothing to do with JSX and that could confuse people.

**Comment by svenjacobs** [comment]
> The problem is that we would need to check the contents of attributes for quotes and switch them if necessary.

You're right, I didn't think about that. However this is already a problem with the current conversion to double quotes. When I write

```
<div style='background-image: url("https://picsum.photos/200/300")'></div>
```

it will be converted by this plugin to the following broken markup

```
<div style="background-image: url("https://picsum.photos/200/300")"></div>
```

I agree that this behaviour should be configured by a new plugin-specific configuration option, not by `singleQuote` or `jsxSingleQuote`.

**Comment by svenjacobs** [comment]
I started to work on this, please see #271. However I need some help. Please have a look at my comment in the PR.
